src/api/sso/test/e2e/signup-flow.test.js

```diff
@@ -117,35 +115,36 @@ describe('Signup Flow', () => {
       expect(jwt.family_name).toEqual(galileoGalilei.lastName);
       expect(jwt.name).toEqual(galileoGalilei.displayName);
       expect(jwt.roles).toEqual(['seneca', 'telescope']);
-      expect(jwt.picture).toEqual(galileoGalilei.github.avatarUrl);
+      expect(jwt.picture).toEqual(galileoGalilei.githubAvatarUrl);
 
       return { id: jwt.sub, token };
     }
 
-    // Part 4: logout so we can try logging in again as a registered user.
-    // Confirm that the token payload matches our upgraded user status.
-    // Confirm that the data in the Users service for this user
-    // matches what we expect, and that our token allows us to access it.
-    async function partFour(id, token) {
-      // Logout
-      await logout(page);
-
-      // Use this upgraded token to get our user profile info and confirm.
-      const res = await fetch(`${USERS_URL}/${id}`, {
-        headers: {
-          Authorization: `bearer ${token}`,
-          'Content-Type': 'application/json',
-        },
-      });
-      expect(res.ok).toBe(true);
-      const data = await res.json();
-      expect(data).toEqual(galileoGalilei);
-    }
+    // TODO: I'm not sure what the Supabase equivalent to this should be...
```

Comment:
I don't understand this test, if we have logged out, shouldn't the token have expired?

Reply:
I think the equivalent test here would be using the token to do some request to Supabase as the user vs. using the service token.  We need to make sure that token lets us do things with both the microservices and supabase.